packages/connect-conformance/src/callback-client.ts

```diff
@@ -84,9 +84,34 @@ async function unary(
 
   await wait(req.requestDelayMs);
   return new Promise<ClientResponseResult>((resolve) => {
+    const controller = new AbortController();
+    // Handles client cancellation.
+    controller.signal.addEventListener("abort", () => {
+      resolve(
+        new ClientResponseResult({
+          payloads: payloads,
+          responseHeaders: resHeaders,
+          responseTrailers: resTrailers,
+          error: convertToProtoError(
+            error ?? new ConnectError("operation aborted", Code.Canceled),
+          ),
+        }),
+      );
+    });
+    const { afterCloseSendMs } = getCancelTiming(req);
+    if (afterCloseSendMs >= 0) {
+      wait(afterCloseSendMs)
+        .then(() => controller.abort())
+        .catch(() => {});
```

Comment:
```suggestion
      void wait(afterCloseSendMs).then(() => controller.abort());
```

It doesn't swallow errors, but tells the compiler you're ignoring rejections intentionally.

Reply:
Yes, just an effect of jumping between languages.

Reply:
Yes, just an effect of jumping between languages.